airlock/business_logic.py

```diff
@@ -401,6 +401,16 @@ def output_files_set(self):
             for request_file in filegroup.output_files
         }
 
+    def get_file_review_for_reviewer(self, urlpath: UrlPath, reviewer: str):
```

Comment:
hmm, yeah, the original ticket specified a `get_file_approvals()` like that, which I initially did, and then removed after PR review from Simon https://github.com/opensafely-core/airlock/pull/174#pullrequestreview-1943119900 in favour of keeping the interface between the BLL & DAL this one big call with the entire state of everything. 

Obviously a method that calls down to the DAL would have dodged the issue in tests with stale release_request objects, but refreshing the release_request seems ok in that situation?

Reply:
I think this could be the use case Simon mentioned for just getting a file review without the rest of the release request? Although, I guess we did fetch it all already, and getting the release request already extracted all the reviews from the database, so this avoids another db call